libs/libcommon/tests/test_croissant_utils.py

```diff
@@ -30,6 +34,21 @@ def test_truncate_features_from_croissant_crumbs_response(num_columns: int) -> N
         assert "max number of columns reached" in content["recordSet"][0]["description"]
 
 
+@pytest.mark.parametrize(
+    "feature_name, expected_output",
+    [
+        ("simple_feature", "simple_feature"),
+        ("feature/with/slash", "['feature/with/slash']"),
+        ("feature'with'quote", r"['feature\'with\'quote']"),
+        ("feature[with]brackets", r"['feature\[with\]brackets']"),
+        ("feature[with/slash]'and'quote", r"['feature\[with/slash\]\'and\'quote']"),
```

Comment:
Can you add a test case when the feature_name already contains an escaped character? (To see how it reacts)

Reply:
Done, thanks!